sns_ik_examples/src/test_sns_ik.cpp

```diff
@@ -1,104 +0,0 @@
-/* test for the IK library*/
```

Comment:
Would there be any value in keeping this test around? Though, to be honest, I'm not sure it even still runs...